lib/collective.lib.js

```diff
@@ -176,3 +176,15 @@ export const canContributeRecurring = (collective, user) => {
     ),
   );
 };
+
+/*
+ * Displays the name string as "Legal name (Display name)" if legal name exists.
+ * Example: Sudharaka (Suds)
+ */
+export const concatenateDisplayNameWithLegalName = (legalName, displayName) => {
+  if (legalName) {
```

Comment:
As per https://github.com/opencollective/opencollective-rest/pull/390#discussion_r712880863

```suggestion
  if (legalName && legalName !== displayName) {
```

Reply:
Done. 👍🏽